presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -0,0 +1,96 @@
+===========================
+Presto Native - Prestissimo
+===========================
+
+Prestissimo is a C++ drop-in replacement for Presto workers based on the Velox
+library. It implements the same RESTful API as Java workers using the Proxygen
```

Comment:
```suggestion
library. It implements the same RESTful endpoints as Java workers using the Proxygen
```

I would avoid using the word API because I would argue Presto doesn't have an API, it has an SPI.